MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+graft frontend/templates
```

Comment:
Is MANIFEST from Flask or your IDE, if IDE remove this file

Reply:
Flask required file

Reply:
🐒